react-todo-app/src/components/TodoItem.js

```diff
@@ -35,6 +35,11 @@ if (state.editing) {
 } else {
   editMode.display = "none"
 }
+useEffect(() => {
+  return () => {
+    console.log("Cleaning up...")
```

Comment:
- [ ] Kindly consider removing console.log from your codebase as it could make ur code more professional.